### PR TITLE
Web10677 color-white and color-black referencing colours correctly

### PR DESCRIPTION
**Pull request by jask84**
#### What does this PR do? (please provide any background)
https://hxshortbreaks.atlassian.net/browse/WEB-10677
Very small changes so that color-white and color-black reference the correct colours for a few brands. Shouldn't have made any actual colour changes.

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
none as such. Haven't looked at all files for all the brands.

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [x] :+1:
- [x] UXUI Team :+1:
- [x] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

**Comment by stuartgiles** [comment]
While this will work. There is no need for all the duplication.
A better way would probably be to create `@colour-white: rgb(255,255,255);` and `@colour-black: rgb(0,0,0);` in `globals`. Then removing any instance of `@white` or `@black` from all brands and apps.

**Comment by stuartgiles** [comment]
Just spoke to @nassimdonald.
Another option is to do this in `globals`
```
@colour-white: rgb(255,255,255);
@colour-black: rgb(0,0,0);
@white:  @colour-white;
@black: @colour-black;
```
Then remove all references in the brands.

This allows us to continue supporting any apps that use either method.

We should discourage the use of `@white` and `@black` and eventually remove it.

**Comment by jask84** [comment]
Thanks for the comments @stuartgiles. I completely agree and will go with the second option as it makes sense to prevent duplication and just in case any apps use either method.

File: brands/globals/src/less/colours.less [hunk 1]
```diff
@@ -1,5 +1,7 @@
-@white: white;
-@black: black;
+@color-white: rgb(255,255,255);
+@color-black: rgb(0,0,0);
+@white: @color-white;
```
Comment (nassimdonald) [inline brands/globals/src/less/colours.less]: I would write a comment above this line to the effect that these variables are deprecated.

Comment (nassimdonald) [inline brands/globals/src/less/colours.less]: I would add a comment for these old variables to the effect that they are now deprecated.

Comment (jask84) [inline brands/globals/src/less/colours.less]: good point. will do

**Comment by nassimdonald** [comment]
Good to go! 👍

**Pull request merged**